gallery/src/components/NxTag/NxTagPage.tsx

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import {GalleryDescriptionTile, GalleryExampleTile} from '../../gallery-components/GalleryTiles';
+import NxTagExample from './NxTagExample';
+import NxSelectableTagExample from './NxSelectableTagExample';
+import NxTagNarrowExample from './NxTagNarrowExample';
+import { NxTable, NxTableHead, NxTableCell, NxTableRow, NxTableBody } from '@sonatype/react-shared-components';
+
+const NxTagExampleCode = require('!!raw-loader!./NxTagExample').default;
+const NxSelectableTagExampleCode = require('!!raw-loader!./NxSelectableTagExample').default;
+const NxTagNarrowExampleCode = require('!!raw-loader!./NxTagNarrowExample').default;
+
+const NxTagPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <p className="nx-p">
+        There are two types of <code className="nx-code">NxTag</code>. The basic tags have a single state.
+        <code className="nx-code">NxSelectableTag</code> can be selected and deselected. It is up to the consumer
+        what that means in the context of the UI. The are many colors provided for
+        <code className="nx-code">NxTag</code> which can be specified via props.
+      </p>
+      <NxTable>
+        <NxTableHead>
+          <NxTableRow>
+            <NxTableCell>Prop</NxTableCell>
+            <NxTableCell>Type</NxTableCell>
+            <NxTableCell>Required</NxTableCell>
+            <NxTableCell>Default</NxTableCell>
+            <NxTableCell>Details</NxTableCell>
+          </NxTableRow>
+        </NxTableHead>
+        <NxTableBody>
+          <NxTableRow>
+            <NxTableCell>color</NxTableCell>
+            <NxTableCell>
+              'light-blue' | 'purple' | 'pink' | 'blue' | 'red' | 'green' | 'orange' | 'yellow' | 'lime' | 'indigo'
+            </NxTableCell>
+            <NxTableCell>No</NxTableCell>
+            <NxTableCell></NxTableCell>
+            <NxTableCell>
+              If no `color` is specified then the default (dark blue with a grey unselected state) colors are used.
```

Comment:
Use `<code>` instead of backticks

Reply:
> Use `<code>` instead of backticks

https://github.com/sonatype/sonatype-react-shared-components/pull/290/commits/1c7b8448a417f1cf0b22c4a498cc6781b39fc8bf